src/pages/DatasetCatalog.js

```diff
@@ -438,18 +438,18 @@ class DatasetCatalog extends Component {
                               p({ isRendered: property.propertyName !== 'dbGAP' }, [
                                 span({ id: trIndex + "_datasetName", name: "datasetName", isRendered: property.propertyName === 'Dataset Name' }),
                                 span({ id: trIndex + "_datasetId", name: "datasetId", isRendered: property.propertyName === 'Dataset ID' }),
-                                span({ id: trIndex + "_dataType", name: "dataType", isRendered: property.propertyName === 'Data Type' }),
-                                span({ id: trIndex + "_species", name: "species", isRendered: property.propertyName === 'Species' }),
-                                span({ id: trIndex + "_phenotype", name: "phenotype", isRendered: property.propertyName === 'Phenotype/Indication' }),
                                 span({ id: trIndex + "_participants", name: "participants", isRendered: property.propertyName === '# of participants' }),
+                                span({ id: trIndex + "_phenotype", name: "phenotype", isRendered: property.propertyName === 'Phenotype/Indication' }),
                                 span({ id: trIndex + "_description", name: "description", isRendered: property.propertyName === 'Description' }),
+                                span({ id: trIndex + "_dataType", name: "dataType", isRendered: property.propertyName === 'Data Type' }),
+                                span({ id: trIndex + "_species", name: "species", isRendered: property.propertyName === 'Species' }),
                                 property.propertyValue
                               ]),
 
                               a({
                                 id: trIndex + "_linkdbGap",
                                 name: "link_dbGap",
-                                isRendered: property.propertyName === 'dbGAP',
+                                isRendered: property.propertyName === 'Repository',
```

Comment:
This looks like a bug. Without the right property name here, the dbgap link doesn't display.